### ISSUE TITLE
Change the Main#index route to point to the ToDos#index action 

### ISSUE THREAD

**Issue by dickenslian** [geddy/geddy#696]
I'm new to geddy. I followed the tutorial to create the todo list app and everything works fine. But when I try to Change the Main#index route to point to the ToDos#index action, I got below error message.
Error
    at createConstructor (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\response\errors.js:68:34)
    at new <anonymous> (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\response\errors.js:79:33)
    at Object.<anonymous> (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\response\errors.js:21:10)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\controller\responder\index.js:6:14)

Here is what I do to fulfill the requirement:
router.get('/').to('ToDos.index');

I don't know what is going wrong. I try to google it for quite some time but fail to find a solution. Please advice!

**Comment by phanect** [comment]
Current version of Geddy has bug on error handling. Error message and stacktrace are not printed correctly.

I fixed it before and I believe the patch is included current master.
Install master branch and you will see appropriate error message.

If my patch is not included in master, use v13 or v14 branch, or apply following patch manually.
https://github.com/geddy/geddy/pull/673

**Comment by dickenslian** [comment]
Thank you for your answer Phanect. 

I replace "errorConstructor.prototype = new Error();" with "errorConstructor.prototype = Object.create(Error.prototype);" in the errors.js and now I get below error message, which is more helpful.

/ not found.
    at handleNotFound (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\app\index.js:169:11)
    at handleStaticFile (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\app\index.js:137:14)
    at handleNoMatchedRoute (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\app\index.js:185:12)
    at Domain.<anonymous> (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\app\index.js:336:16)
    at Domain.run (domain.js:197:16)
    at Server.<anonymous> (C:\Users\lianhr\AppData\Roaming\npm\node_modules\geddy\lib\app\index.js:282:11)
    at Server.emit (events.js:110:17)
    at HTTPParser.parserOnIncoming [as onIncoming] (_http_server.js:491:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:23)
    at Socket.socketOnData (_http_server.js:343:22)

Basically, I just want to change the default website page to ToDos's index page. Do I need to change any other file besides "router.js"? What is the correct way to fulfill that?

**Comment by phanect** [comment]
I think routing is not configured correctly too. Can you show me your router.js?

**Comment by phanect** [comment]
How about

```javascript
router.get('/').to('to_dos.index');
```

if your ToDo controller is in app/controllers/to_dos.js

You need to configure file name, not controller variable name.
